extension/src/components/ChannelSelection/components/Search/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import SearchInput from 'react/components/UI/SearchInput';
+
+class Search extends PureComponent {
+  static propTypes = {
+    onChange: PropTypes.func.isRequired,
+  }
+
+  render() {
+    const { onChange } = this.props;
+
+    return (
+      <SearchInput
+        onQueryChange={onChange}
+        placeholder="Filter channels"
+      />
+    );
```

Comment:
Something is wrong here the icon isn't being sized correctly; probably because of ems.